runtime/lua/vim/loader.lua

```diff
@@ -41,6 +43,16 @@ local Loader = {
   },
 }
 
+--- @param path string
+--- @return CacheHash
```

Comment:
nitpick, but you should probably return `uv_fs_t` here instead since that's what it is.
Might be useful later for other functionality.

The loader only needs the `CacheHash` which is a subset of `uv_fs_t`